src/components/TextField/TextField.tsx

```diff
@@ -33,7 +29,7 @@ export type Props = TextFieldProps & { label?: string };
  * @todo params
  */
 export default React.forwardRef<HTMLDivElement, Props>((props, ref) => {
-    const { children, label, name, id, ...passThroughProps } = props;
+    const { children, label, name, id, type, ...passThroughProps } = props;
```

Comment:
I removed this since I am not using it, but this was because I had trouble validating email even when the type specified was email.
In one of my attempts, I tried adding it there to "force it" to use the passed in type, this did not work so I just used the regex from another endpoint to check it.